device-types/HPE/HP-6200-24G-mGBIC-yl.yml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: HPE
+model: HP 6200-24G-mGBIC yl
```

Comment:
Is this a ProCurve model? if so it should have ProCurve in the model like other HPE ProCurve devices in this repository